whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceTestIntegration.java

```diff
@@ -374,6 +374,7 @@ public void lookup_org_single_language_codes() {
                 "geoloc:         52.375599 4.899902\n" +
                 "language:       DK\n" +
                 "admin-c:        TP1-TEST\n" +
+                "country:        NL\n"+
```

Comment:
Can you also add a test to `lookup_org_single_language_codes` and `lookup_org_multiple_language_codes` that country is set properly in the organisation `Entity` response

Reply:
There is already one test for lookup_org_multiple_language_codes 
I will create lookup_org_single_language_codes_no_redaction

Reply:
"country" is only specified in IP and aut-num objects, not in entities, so we can't test for it there.